ge/org.osate.ge.ba/src/org/osate/ge/ba/BehaviorAnnexBusinessObjectProvider.java

```diff
@@ -49,6 +52,9 @@
 			final BehaviorAnnex ba = (BehaviorAnnex) bo;
 			return Stream.concat(Stream.concat(ba.getTransitions().stream(), ba.getStates().stream()),
 					ba.getVariables().stream());
+		} else if (bo instanceof BehaviorTransition) {
+			final BehaviorTransition bt = (BehaviorTransition) bo;
+			return getDispatchCondition(bt);
```

Comment:
No need for a separate function or using streams, filtering, and mapping. Just keep it simple.  Just return `Stream.of(new DispatchCondition())` if the transition has a condition that can be shown.  The current check is checking if bt is null which is never the case.